### ISSUE TITLE
spurious(?) traceback on import of folder as new project

### ISSUE THREAD

**Issue by planetmaker** [spyder-ide/spyder#6421]
## Description

**What steps will reproduce the problem?**

1. Try to import a folder with many sub-folders, many non-python-files, as project, total size 1.3G including mercurial repository data
2. 
3. 

**What is the expected output? What do you see instead?**
Folder imported as project (hey, it was the first thing I tried in this respect) - or a message what failed. However... things still seem to work.

**Please provide any additional information below**

  File "/home/ingo/anaconda3/lib/python3.6/site-packages/spyder/plugins/projects.py", line 148, in <lambda>
    lambda v: self.editor.setup_open_files())
  File "/home/ingo/anaconda3/lib/python3.6/site-packages/spyder/plugins/editor.py", line 2662, in setup_open_files
    self.close_all_files()
  File "/home/ingo/anaconda3/lib/python3.6/site-packages/spyder/plugins/editor.py", line 1974, in close_all_files
    self.editorstacks[0].close_all_files()
  File "/home/ingo/anaconda3/lib/python3.6/site-packages/spyder/widgets/editor.py", line 1408, in close_all_files
    while self.close_file():
  File "/home/ingo/anaconda3/lib/python3.6/site-packages/spyder/widgets/editor.py", line 1370, in close_file
    self.remove_from_data(index)
  File "/home/ingo/anaconda3/lib/python3.6/site-packages/spyder/widgets/editor.py", line 1163, in remove_from_data
    self.update_fileswitcher_dlg()
  File "/home/ingo/anaconda3/lib/python3.6/site-packages/spyder/widgets/editor.py", line 822, in update_fileswitcher_dlg
    self.fileswitcher_dlg.setup()
  File "/home/ingo/anaconda3/lib/python3.6/site-packages/spyder/widgets/fileswitcher.py", line 757, in setup
    current_path = self.current_path
  File "/home/ingo/anaconda3/lib/python3.6/site-packages/spyder/widgets/fileswitcher.py", line 346, in current_path
    return self.paths_by_widget[self.get_widget()]
KeyError: None

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.2
* Qt Versions: 5.6.2, PyQt5 5.6 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.12.1 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
psutil >=0.3     :  5.2.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.2 (OK)

```

**Comment by ccordoba12** [comment]
I think this is fixed in our latest release (3.2.6). Please update and try again.

**Comment by CAM-Gerlach** [comment]
Looks like a direct duplicate of #5317 , which was fixed in 3.2.4. You can update with ``conda update spyder``. Therefore, this will be closed. If updating doesn't fix it, then please comment and re-open this, describing in detail the circumstances in which the error occurred, and post the information listed in [relevant subsection](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ#must-have-and-helpful-items-to-include) of the Calling for Help section as a comment to this issue, and we'll try to help if we can.
